package.json

```diff
@@ -10,14 +10,15 @@
     "url": "https://github.com/ipython/ipywidgets.git"
   },
   "scripts": {
+    "postinstall": "npm run bower",
```

Comment:
I agree this is needed, because we don't package bower contents, but I don't think the changes to prepublish and publish are needed.